### ISSUE TITLE
Add tutorial/User Manual for the interactive display

### ISSUE THREAD

**Issue by michoug** [merenlab/anvio#433]
Hi,
Sorry if it already exists, but I found the interactive display options a bit unclear.
I would suggest to add a tutorial page on the different options to increase the efficiency of the users.
Best

Greg

**Comment by meren** [comment]
Hi, Greg. What do you mean by _interactive display options_?

Is it what is shown on the left side of the display? Menu items, etc?

**Comment by michoug** [comment]
Hi,
Yes, i was trying to improve the figure, as some samples names were cut and some of the options didn't seems very clear to me, such as the tick box near the different layers, and others.
Best
Greg

**Comment by meren** [comment]
Fair enough :) I guess we should shoot a video while we work on a dataset to highlight some best-practices.
